test/integration/test.js

```diff
@@ -0,0 +1,79 @@
+var assert = require('assert');
+var server = require('blueoak-server');
+var fs = require('fs');
+var testUtil = server.testUtility();
+global.__appDir = __dirname;
+
+/**
+ * You MUST have couchdb running locally with a database called 'basic' in order for these tests to work
+ */
```

Comment:
also, this file should be called something more specific, like `testDesignDocumentUpdates.js`